Atomic/containers.py

```diff
@@ -289,7 +290,72 @@ def _mark_vulnerable(self, containers):
             if con.id in vulnerable_uuids:
                 con.vulnerable = True
 
+    def update_all_containers(self):
+        could_not_pull = {}
+        preupdate_containers = self.syscontainers.get_containers()
+        for i in preupdate_containers:
+            img = i['Image']
+            pull_location = i['PullLocation']
+            name = i['Names'][0]
+            d = util.Decompose(img)
+            if d.registry == "":
+                util.write_err("Image {} not fully qualified: skipping {}".format(img, name))
+                continue
+            try:
+                self.syscontainers.pull_image(pull_location)
+            except:  # pylint: disable=bare-except
+                could_not_pull[img] = True
+                util.write_err("Image {} could not be pulled.  You may need to manually update {}".format(img, name))
+                continue
+
+            self.syscontainers.update_container(name, controlled=True)
+        postupdate_containers = self.syscontainers.get_containers()
+
+        images_by_name = {i['RepoTags'][0] : i for i in self.syscontainers.get_system_images()}
+        preupdate_containers_by_name = {i['Names'][0] : i for i in preupdate_containers}
+        postupdate_containers_by_name = {i['Names'][0] : i for i in postupdate_containers}
+
+        def get_image_id(c): return c['ImageId'] if 'ImageId' in c else c['ImageID']
```

Comment:
hmm is `ImageId` included in container part? I don't think I saw it in the code base. 
https://github.com/projectatomic/atomic/blob/master/Atomic/syscontainers.py#L1308,
seems like only ImageID was there?